packages/widgetbook_cli/bin/commands/publish.dart

```diff
@@ -479,19 +441,6 @@ class PublishCommand extends WidgetbookCommand {
           progress.complete('Uploaded build');
         }
 
-        // If generator is not run or not properly configured
-        if (themes.isEmpty) {
-          logger.err(
-            'HINT: Could not find generator files. '
-            'Therefore, no review has been created. '
-            'Make sure to use widgetbook_generator and '
-            'run build_runner before this CLI. '
-            'See https://docs.widgetbook.io/widgetbook-cloud/review for more '
-            'information.',
-          );
-          throw ReviewNotFoundException();
-        }
-
```

Comment:
I think it would be good to keep this in. Obviously this won't be possible with themes, but you can use the usecases instead.